pkg/driver/node_test.go

```diff
@@ -1924,6 +1925,7 @@ func TestNodeGetInfo(t *testing.T) {
 			instanceID:        "i-123456789abcdef01",
 			instanceType:      "t2.medium",
 			availabilityZone:  "us-west-2b",
+			region:            "us-west-2b",
```

Comment:
`"us-west-2"` is a region. `"us-west-2b"` is not.

Reply:
Ack, made changes.